cai_causal_graph/time_series_causal_graph.py

```diff
@@ -275,8 +276,8 @@ def get_minimal_graph(self) -> TimeSeriesCausalGraph:
                     edge = self._EdgeCls(
                         source=source, destination=destination, edge_type=edge.edge_type, meta=edge.meta
                     )
-
-                    minimal_cg.add_edge(edge=edge)
+                    # Do not check for acyclicity as we know the original graph was a DAG, so this will be a DAG
```

Comment:
Do we definitely know this? Should we check it is a DAG? One can make a TS graph that is not a DAG.